managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -492,16 +493,24 @@ export default class ClusterFields extends Component {
       this.props.fetchUniverseResources(nextProps.universe.universeConfigTemplate.data);
     }
     // If nodeInstanceList changes, fetch number of available nodes
-    if (
-      getPromiseState(nodeInstanceList).isSuccess() &&
-      getPromiseState(this.props.cloud.nodeInstanceList).isLoading()
-    ) {
-      let numNodesAvailable = nodeInstanceList.data.reduce((acc, val) => {
+    if (getPromiseState(nodeInstanceList).isSuccess()) {
+      let nodesPerRegion = [];
```

Comment:
should be `const` as it doesn't reassign anywhere

Reply:
Done.